site/_guides/deploy-aws-nlb.md

```diff
@@ -12,22 +12,26 @@ This configuration has several advantages:
 
 ## Moving parts
 
-- We run Contour as a DaemonSet across the cluster.
-- The Contour pod runs with host networking and binds to port 8080 and 8443 on the node.
-- Host networking means that traffic hits Envoy without transitioning through any other fancy networking hops.
+- We run Contour as a DaemonSet across the cluster
+- The Contour pod runs with host networking and binds to port 80 and 443 on the node
```

Comment:
We use `hostPort`, not `hostNetwork`.  Probably should clarify here and below.

Reply:
gotcha, also realized it still says we run Contour as a daemonset which is incorrect. =)